sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -151,6 +153,50 @@ public static SentryEnvelopeItem fromUserFeedback(
     return new SentryEnvelopeItem(itemHeader, () -> cachedItem.getBytes());
   }
 
+  public static SentryEnvelopeItem fromAttachment(
+      @NotNull ILogger logger, final @NotNull Attachment attachment) {
+
+    final CachedItem cachedItem =
+        new CachedItem(
+            () -> {
+              if (attachment.getBytes() != null) {
+                return attachment.getBytes();
+              } else if (attachment.getPathname() != null) {
+                try (FileInputStream fileInputStream =
+                        new FileInputStream(attachment.getPathname());
+                    BufferedInputStream inputStream = new BufferedInputStream(fileInputStream);
+                    ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```

Comment:
H: we could still wrap the outputStream to a buffered writer, see `fromSession` example

Reply:
As discussed on Friday, it doesn't really make sense to use a buffered writer, which is a character-based output stream, as we are dealing with bytes here and not characters as we do in `fromSession`.

Reply:
if you both agreed, it's fine, I was not part of the discussion